docs/wallets/ledger-wallet.mdx

```diff
@@ -19,4 +19,215 @@ for:
 - Transferring tokens
 - Signing transactions
 
+:::important
+
 At this time, Hotspot management actions such as onboarding or asserting location are not supported.
+
+:::
+
+## How To Access Ledger Account in Helium Mobile Wallet
+
+So let's take a look at how to access your Ledger account via the Helium Mobile Wallet app.
+
+### Step 1: Preparation
+Before you begin, there are prerequsites and software you need to install which includes:
+
+* Helium Ledger Nano X Device
+* [Ledger Live Application](https://www.ledger.com/ledger-live)
+* Helium Mobile Wallet app
+
+Nano X users need to have Bluetooth connectivity to manage HNT using the app.
+
+### Step 2: Install the Solana app on your Ledger device
+
+:::important
+
+Helium migration to Solana
+
+Helium blockchain migrated to Solana blockchain. The migration to Solana took place on April 18, 2023, beginning at 9am PST. After the migration the Helium blockchain is officially halted. Learn more
+
+As a result, the Helium app was removed from Ledger Live. After the migration, users should be able to manage their HNT tokens with the Solana app. 
+
+:::
+
+To do this, first open "Ledger Live" and select "Settings" > Experimental features and activate "Developer mode". Then connect your Ledger device to "My Ledger" to ensure that your device is running the latest firmware version. If you can not find "Solana", you may need to update your Ledger Live. So check your version of Ledger Live by navigating to Settings, and select About and Version.
+
+<figure className="screensnippet-wrapper">
+  <img
+    src={useBaseUrl('/img/wallets/ledger/install-app.png')}
+    className="add-shadow add-shadow-margin add-border-radius"
+    style={{ maxHeight: 'initial', maxWidth: '720px' }}
+  />
+  <figcaption>Add Solana App".</figcaption>
+</figure>
+<br />
+
+Once you have done this, click on "My Ledger" and install the "Solana" app to your Ledger device.
+
+You have to ensure your Ledger Live app and Ledger device are running on the latest firmware. If your Ledger Live is out of date, you will likely see update notifications appear at the top of the window. Otherwise, you can manually check your version of Ledger Live by navigating to Settings, select About and Version. You will find the "update" button.
+
+### Step 3: Pair your Ledger with the Helium Mobile Wallet app (Nano X only)
+
+Since this process requires Bluetooth connectivity, go to the "Control Center" of your Nano X device and ensure that the Bluetooth is on. If you have any issues owith Bluetooth, watch [this video](https://www.youtube.com/watch?v=wHUkE1Gw0hs) for possible solutions. Then use the buttons on your Ledger device to open the Solana app. Now, open the Helium Mobile Wallet app on your phone and select "Add New Wallet".
```

Comment:
```suggestion
Since this process requires Bluetooth connectivity, go to the "Control Center" of your Nano X device and ensure that the Bluetooth is on. If you have any issues with Bluetooth, watch [this video](https://www.youtube.com/watch?v=wHUkE1Gw0hs) for possible solutions. Then use the buttons on your Ledger device to open the Solana app. Now, open the Helium Mobile Wallet app on your phone and select "Add New Wallet".
```